examples/resources/mutation_policy/resource_cluster_scoped_annotation_mutation_policy.tf

```diff
@@ -31,7 +31,7 @@ resource "tanzu-mission-control_mutation_policy" "cluster_annotation_mutation_po
         {
           key      = "component"
           operator = "NotIn"
-          values = [
+          values   = [
```

Comment:
In the previous MR, there were many files with such formatting issues.

Reply:
Right, I have tried to auto-format all .tf files, but that has created too many differences. All fields you have mentioned are located in the same folder.
I have formatted them to bring back those differences.